### ISSUE TITLE
make latexpdf fails

### ISSUE THREAD

**Issue by ywwg** [mixxxdj/manual#40]
I checked out manual HEAD and tried to make a pdf, and got:

! Package pdftex.def Error: File `chapters/../_static/Mixxx-112-LateNight-Effects-Deck.png' not found.

Did someone forget to git add some files?

**Comment by kain88-de** [comment]
@esbrandt wanted to add the pictures once before the release. To ensure that all the picture come from one OS and the interface looks similar.

This should only affect the current master and not older versions of the manual.

**Comment by ywwg** [comment]
Now that code is basically complete, I'd like to move forward on this.  It should be safe to grab screenshots without them getting out of date

**Comment by kain88-de** [comment]
Yeah but they should all come from one machine to look consistent. I'm not sure how much time @esbrandt has to do this. But it should be OK if you replace them all at once with a linux system.

**Comment by mxmilkiib** [comment]
resolved;

"the build-pdf CI job runs a real sphinx-build -b latex + latexmk build that passes on every PR and 2.5 push"
